src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -269,9 +269,27 @@ static bool filterConversion(Conversion conversion)
                     ensureAllUnderlyingConversionsChecked(syntax, source, conversion, wasCompilerGenerated, destination, diagnostics);
                 }
 
+                var boundSource = BindToNaturalType(source, diagnostics);
```

Comment:
>var boundSource = BindToNaturalType(source, diagnostics); [](http://example.com/codeflow?start=16&length=57)

Is doing this before the check makes a difference? It looks like we wouldn't classify conversion as ```ImplicitPointer``` unless source is already known to be a function pointer